src/components/IconHeaderButton.tsx

```diff
@@ -6,7 +6,7 @@ import Colors from '../themes/colors'
 
 const styles = StyleSheet.create( {
   native: {
-    margin: -16,
+    margin: 0,
```

Comment:
Honestly, I wondered if you'd notice this. Glad you caught it. I wanted to delete it all, but then for some reason hesitated because it looked like a big change, and I figured it'd be harder to review. I cleaned it up now.